gravitee-apim-console-webui/src/services-ngx/notification-settings.service.ts

```diff
@@ -33,4 +33,12 @@ export class NotificationSettingsService {
   delete(apiId: string, id: string): Observable<NotificationSettings[]> {
     return this.http.delete<NotificationSettings[]>(`${this.constants.env.baseURL}/apis/${apiId}/notificationsettings/${id}`);
   }
+
+  getNotifiers(apiId: string): Observable<any> {
+    return this.http.get(`${this.constants.env.baseURL}/apis/${apiId}/notifiers`);
+  }
+
+  create(apiId, notificationConfig): Observable<any> {
```

Comment:
Same some ts type & test missing here ;) 

notificationConfig should be NewNotification type IMHO

Reply:
Done